modules/space/rendering/renderableconstellationbounds.cpp

```diff
@@ -192,10 +154,13 @@ void RenderableConstellationBounds::deinitializeGL() {
 }
 
 bool RenderableConstellationBounds::isReady() const {
-    return (_vao != 0) && (_vbo != 0) && _program;
+    if (!_hasLabel) {
```

Comment:
How about refactoring this function to avoid repeating the long check?
Maybe something like: 
```
bool isReady = _program && _vao != 0 && _vbo != 0;
if (_hasLabel) {
    isReady = isReady && !_labelset.entries.empty(); // or &= version, depending on how you like that one
}
return isReady;
```